tests/FSharp.Compiler.ComponentTests/ErrorMessages/ConstructorTests.fs

```diff
@@ -1,41 +1,39 @@
 // Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
 
-namespace FSharp.Compiler.UnitTests
+namespace FSharp.Compiler.ErrorMessages.ComponentTests
 
-open NUnit.Framework
+open Xunit
 open FSharp.Test.Utilities
 open FSharp.Compiler.SourceCodeServices
 
-[<TestFixture>]
 module ``Constructor`` =
 
-    [<Test>]
+    [<Fact>]
     let ``Invalid Record``() =
         CompilerAssert.TypeCheckWithErrors
             """
 type Record = {field1:int; field2:int}
 let doSomething (xs) = List.map (fun {field1=x} -> x) xs
-
 doSomething {Record.field1=0; field2=0}
             """
             [|
-                FSharpErrorSeverity.Error, 1, (5, 13, 5, 40), "This expression was expected to have type\n    'Record list'    \nbut here has type\n    'Record'    "
-                FSharpErrorSeverity.Warning, 20, (5, 1, 5, 40), "The result of this expression has type 'int list' and is implicitly ignored. Consider using 'ignore' to discard this value explicitly, e.g. 'expr |> ignore', or 'let' to bind the result to a name, e.g. 'let result = expr'."
+                FSharpErrorSeverity.Error, 1, (4, 13, 4, 40), "This expression was expected to have type\n    'Record list'    \nbut here has type\n    'Record'    "
+                FSharpErrorSeverity.Warning, 20, (4, 1, 4, 40), "The result of this expression has type 'int list' and is implicitly ignored. Consider using 'ignore' to discard this value explicitly, e.g. 'expr |> ignore', or 'let' to bind the result to a name, e.g. 'let result = expr'."
             |]
 
-    [<Test>]
+    [<Fact>]
     let ``Comma In Rec Ctor``() =
         CompilerAssert.TypeCheckWithErrors
             """
 type Person = { Name : string; Age : int; City : string }
 let x = { Name = "Isaac", Age = 21, City = "London" }
             """
             [|
-                FSharpErrorSeverity.Error, 1, (3, 18, 3, 52), "This expression was expected to have type\n    'string'    \nbut here has type\n    ''a * 'b * 'c'    \r\nA ';' is used to separate field values in records. Consider replacing ',' with ';'."
+                FSharpErrorSeverity.Error, 1, (3, 18, 3, 52), "This expression was expected to have type\n    'string'    \nbut here has type\n    ''a * 'b * 'c'    " + System.Environment.NewLine + "A ';' is used to separate field values in records. Consider replacing ',' with ';'."
```

Comment:
System.Environment.Newline perhaps?  There are many other places too.  Maybe \n is correct.

Reply:
I’ve noticed that it is mixed all over the place. I usually test it on both windows and Linux and use the NewLine where applicable.